templates/webhook.bin.erb

```diff
@@ -154,6 +154,15 @@ class Server < Sinatra::Base
         data['repository']['default_branch']                     # github tagged release; no ref.
       ).sub('refs/heads/', '') rescue nil
 
+    deleted = (
+      data['deleted'] || # Github
+      data['after']      # Github and Gitlab
+    )
+
+    if deleted == true || deleted == '0000000000000000000000000000000000000000'
+      branch_deleted = true
```

Comment:
maybe a `deleted?` method that could be called from `deploy` would be better??

Reply:
After looking into building a `delete?` I realized that it would be difficult without passing the payload data from the endpoint through the `deploy` method into the `delete?` method (at least based on my limited understanding of Sinatra). 

Definitely seems like something we should do, but may be better to wait for a larger refactor of the code.

Reply:
Line 180 `deleted = data['delered'] rescue false` can go now though?

Reply:
Hmm, not sure how I missed that. We'll actually need some of that. I'll update the PR so that block can use the values from gitlab and bitbucket.